src/CellData/SkeletonCellFieldPair.jl

```diff
@@ -0,0 +1,98 @@
+"""
+  SkeletonCellFieldPair is a special construct for allowing uh.plus
+  and uh.minus to be two different CellFields. In particular, it is
+  useful when we need one of the CellFields to be the dualized
+  version of the other for performing ForwardDiff AD of a Skeleton
+  integration DomainContribution wrt to the degrees of freedom
+  of the CellField, plus and minus sensitivities done separately,
+  so as to restrict the interaction between the dual numbers.
+
+  It takes in two CellFields and stores plus version of CellFieldAt
+  for the first CellField and minus version of CellFieldAt for the
+  second the CellField. SkeletonCellFieldPair is associated with the
```

Comment:
Remove first "the"

Reply:
Perhaps we may mention that we conceive SkeletonCellFieldPair as in internal code artifact of Gridap and we do not pretend the user to deal with it.

Reply:
Done!